src/test/java/uk/gov/companieshouse/overseasentitiesapi/validation/OverseasEntitySubmissionDtoValidatorTest.java

```diff
@@ -322,7 +322,7 @@ void testEntityNameFieldValidatorGetsCalled() {
         verify(presenterDtoValidator, times(0)).validate(any(), any(), any());
         verify(entityDtoValidator, times(0)).validate(any(), any(), any());
         verify(dueDiligenceDataBlockValidator, times(1)).validatePartialDueDiligenceFields(any(), any(), any(), any());
-        verify(trustDetailsValidator, times(0)).validate(any(), any(), any());
+        verify(trustDetailsValidator, times(0)).validate(any(), any(), any(), eq(true));
```

Comment:
validatePartial is being called in this test so not sure why isFullValidation would be true here.

Reply:
Good spot. It shouldn't be called at all, with either a true or a false value. Will fix.